Star Trek Adventurers 2d20/Star Trek Adventures 2d20.css

```diff
@@ -0,0 +1,162 @@
+/* === GENERAL === */
+.charsheet {
+    min-width: 820px;
+    background: #FFFFFF;
+    background-image: url("https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/Star Trek Adventurers 2d20/Sheet_blank.PNG");
```

Comment:
I noticed the other 2d20 sheets are using this, the files should be static, but what do you guys think is best practice, I can toss it up to imgur if that is better.